### ISSUE TITLE
Silverblue Surface Pro 4 unable to install "Dummy Kernel"

### ISSUE THREAD

**Issue by jkrogers9862** [linux-surface/linux-surface#1511]
Following the steps laid out in the instruction guide seems no longer possible given the "dummy" kernel provided for the Silverblue edition of Fedora. It stops in the post post-transactions with a error. As have not saved the logs so I will try again to gather them but I want to get this on the radar. 

Expected result: following the instruction steps to override replace removes the old kernel and replaces it with kernel-surface

Actual result: rpm-ostree override replace fails in the post transactions stage.

Possible OBE solutions: the first time I tried, I had create the boot partition manually with out 600 MiB of space, then I increased this to 1000 MiB and the error message changed. Given this it is possible that way too much space is being taken up to install this kernel on the system.

**Comment by LorbusChris** [comment]
I hit this problem on a Surface Pro 7+ w/ Fedora 40 Silverblue, however it's not caused by the dummy package but it occurs when `kernel-surface` is installed on a system with an insufficient EFI partition size. I doubled both the EFI and the boot partition sizes to 1200 MiB and 2048 MiB, respectively, and it succeeded. The boot partition size increase might not be needed, but I won't try again since configuring the partitions manually in the Anaconda installer is pretty finicky.

The dummy package isn't needed at all, and the docs should be updated to this simpler invocation (I'll open a PR):
```bash
rpm-ostree override remove kernel kernel-core \
                           kernel-modules kernel-modules-extra \
                           libwacom libwacom-data \
                           --install kernel-surface \
                           --install iptsd \
                           --install libwacom-surface \
                           --install libwacom-surface-data
```

**Comment by jkrogers9862** [comment]
> I hit this problem on a Surface Pro 7+ w/ Fedora 40 Silverblue, however it's not caused by the dummy package but it occurs when `kernel-surface` is installed on a system with an insufficient EFI partition size. I doubled both the EFI and the boot partition sizes to 1200 MiB and 2048 MiB, respectively, and it succeeded. The boot partition size increase might not be needed, but I won't try again since configuring the partitions manually in the Anaconda installer is pretty finicky.
> 
> The dummy package isn't needed at all, and the docs should be updated to this simpler invocation (I'll open a PR):
> 
> ```shell
> rpm-ostree override remove kernel kernel-core \
>                            kernel-modules kernel-modules-extra \
>                            libwacom libwacom-data \
>                            --install kernel-surface \
>                            --install iptsd \
>                            --install libwacom-surface \
>                            --install libwacom-surface-data
> ```

I have followed these steps and increated my boot partition to 3 GiB and my EFI to 2.5 GiB and I still get the following error:
`error: Running %posttrans for kernel-surface-core: bwrap(/bin/sh): 하위 프로세스가 1 시그널로 죽었습니다; run `journalctl -t 'rpm-ostree(kernel-surface-core.posttrans)'` for more information`
Running the command results in the following:
`8월 29 13:18:02 fedora rpm-ostree(kernel-surface-core.posttrans)[3780]: error: rpm-ostree kernel-install wrapper: 그런 파일이나 디렉터리가 없습니 >
 8월 29 13:21:55 fedora rpm-ostree(kernel-surface-core.posttrans)[4139]: error: rpm-ostree kernel-install wrapper: 그런 파일이나 디렉터리가 없습니 >`
At least for a Surface Pro 4, there still seems to be an issue to resolve.

**Comment by LorbusChris** [comment]
can you try
```
$ rpm-ostree reset
$ systemctl reboot
$ rpm-ostree update && rpm-ostree override remove kernel kernel-core \
                           kernel-modules kernel-modules-extra \
                           libwacom libwacom-data \
                           --install kernel-surface \
                           --install iptsd \
                           --install libwacom-surface \
                           --install libwacom-surface-data
```

**Comment by jkrogers9862** [comment]
![image](https://github.com/user-attachments/assets/b4ca29c0-40b5-4d28-8cfd-11905bf6fbb7)
Receives the same result. This was a brand new installation, removed all partitions from previous installation.

**Comment by LorbusChris** [comment]
That's too bad, I'm out of ideas now too. Let's rename the issue to `Silverblue Surface Pro 4 unable to install kernel-surface-core` anyway?

**Comment by jkrogers9862** [comment]
For the time being I have rebased to the Bazzite version from ublue which seems to have better out of the box support for the surface pro 4 tablet, enough that I don't need to install this kernel it seems like.

**Comment by LorbusChris** [comment]
> For the time being I have rebased to the Bazzite version from ublue which seems to have better out of the box support for the surface pro 4 tablet, enough that I don't need to install this kernel it seems like.

Interesting, thanks for the hint. Were you able to install `iptsd` on that (I assume, stock Fedora) kernel? Does the touchscreen work?

**Comment by jkrogers9862** [comment]
> > For the time being I have rebased to the Bazzite version from ublue which seems to have better out of the box support for the surface pro 4 tablet, enough that I don't need to install this kernel it seems like.
> 
> Interesting, thanks for the hint. Were you able to install `iptsd` on that (I assume, stock Fedora) kernel? Does the touchscreen work?

Yes, the touchscreen does work with some quirks but it feels very fluid and natural. I was surprised that it just seemed to work but on their website they list the Surface Tablets under the device support list which is what led me to try it.
